osclib/checkrepo.py

```diff
@@ -439,6 +439,12 @@ def check_specs(self, request_id=None, request=None):
         specs = [en.attrib['name'][:-5] for en in root.findall('entry')
                  if en.attrib['name'].endswith('.spec')]
 
+        # special case for glibc.i686, it have not the relevant specfile for glibc.i686
+        # but must be add it to requests list as a dummy request, otherwise the state
+        # has not be check and won't download it's binaries.
+        if 'glibc' in specs:
```

Comment:
yeah, it's a special case, https://build.opensuse.org/package/show/Base:System/glibc.i686 and you can see there is no relevant specfile in glibc what repochecker can recover we've glibc.i686 there, usually it'll be safe if repohcecker did not to check glibc.i686's binaries, BUT when it was together with gcc5, some libraries can not found glibc-32bit as it was generated from glibc.i686.

I'm waiting for glibc.i686 rebuild done in  Base:Ssytem, then I can know will repochecker downloading it's binaries properly, the rest of pre-check in repochecker looks worked.